react/features/toolbox/components/web/Toolbox.js

```diff
@@ -975,7 +980,7 @@ class Toolbox extends Component<Props, State> {
             <RecordButton
                 key = 'record'
                 showLabel = { true } />,
-            this._shouldShowButton('sharedvideo')
+            isModerator && !this._shouldShowButton('sharedvideo')
```

Comment:
Am I missing something or this should be:
```suggestion
            isModerator || this._shouldShowButton('sharedvideo')
```

Reply:
@dincho it should be `and` not `or` because `isModerator` check is required and not optional

Reply:
I assume the requirements are:
Should show the button if the user is moderator or otherwise regular user and it is enabled in the interface config.

Maybe I got that wrong.